src/main/resources/static/app/lab-request/comments.html

```diff
@@ -72,9 +72,9 @@ <h4 class="h4">
                 ng-if="editComment.id">
             {{'Update note'|translate}}
         </button>
-        <div ng-if="isLabUser()">
-        <input type="checkbox" ng-model="editComment.notificationRequested" placeholder="false" id="internal-comment-input">
-        <span><i>{{'Send notification to requester and hub users.' | translate}}</i></span>
+        <div>
+            <input type="checkbox" ng-model="editComment.notificationRequested" placeholder="true" id="internal-comment-input">
+            <span><i>{{'Send notification.' | translate}}</i></span>
```

Comment:
Another option is to have three different descriptions, depending on the user type, so that it is clear to whom the notification is sent (i.e., not to yourself).